var/spack/repos/builtin/packages/jacamar-ci/package.py

```diff
@@ -0,0 +1,49 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import re
+
+from spack.package import *
+
+
+class JacamarCi(Package):
```

Comment:
Is it possible to use GoPackage here?
```suggestion
class JacamarCi(GoPackage):
```
cc: @alecbcs